pixelfusion/docs-internal-processes
# Feature/api deploy

| Q | A |
|--|--|
| Status | READY |
| Migrations | NO |

## Description
Made a `/api` endpoint, now we can setup DatoCMS to hit this endpoint and the content will be fetched from DatoCMS -> build the content with Hugo and after that update DatoCMS with the status of the build.

## Steps to test/reproduce
1. Pull the branch
1. Run `yarn install && composer install`
1. Setup the `.env` file (login to DatoCMS to view the credentials)
1. Bootup a local php server with `php -S localhost:8000`
1. Visit `localhost:8000/api`
1. You should get a json response with `success`

## Todos
[ ] Test if the webhook(datoCMS) works
